### PR TITLE
System updates debian

### PR DESCRIPTION
**Pull request by drewmoseley**
# External Contributor Checklist

🚨 Please review the [guidelines for contributing](https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING.md) to this repository.

- [ ] Make sure that all commits are signed with [`git --signoff`](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work). Also note that the signoff author must match the author of the commit.

### Description

Please describe your pull request.

Thank you!

### CODE REVIEW

**Comment by kacf** [comment]
I merged it for now to get the rest of the updates online, but let me know if you want to do anything about those I pointed out.

File: 03.Devices/02.Yocto-project/03.Raw-flash/docs.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
[the remark this answers is not in the document]
Comment (drewmoseley) [reply 03.Devices/02.Yocto-project/03.Raw-flash/docs.md:1]: Indeed.  I'll push a fix.

File: 04.System-updates-Debian-family/01.Overview/docs.md [hunk 2]
```diff
@@ -78,3 +78,4 @@ Mender has official support for [the Yocto build system](../../System-updates-Yo
 
 For help from the community, as well as links to board integrations, visit [Mender
 Hub](https://hub.mender.io/?target=_blank).
+=======
```
[the remark this answers is not in the document]
Comment (drewmoseley) [reply 04.System-updates-Debian-family/01.Overview/docs.md:81]: Indeed another error.  Eagle eye; well spotted.
I'll remove it.

**Comment by kacf** [comment]
Thanks!

**Pull request merged**